nemo/collections/common/metrics/perf_metrics.py

```diff
@@ -84,7 +84,8 @@ def on_train_end(self, trainer, pl_module):
             logging.error(f"Failed to calculate TFLOPs per sec per GPU.\n{exc}")
 
         logging.info(f"TFLOPs per sec per GPU={tflops_per_sec_per_gpu:.2f}")
-        pl_module.logger.experiment.add_scalar("tflops_per_sec_per_gpu", tflops_per_sec_per_gpu)
+        if pl_module.logger.experiment is not None:
```

Comment:
I think the check should also include `pl_module.logger is not None in case [no logger](https://github.com/Lightning-AI/pytorch-lightning/blob/master/src/lightning/pytorch/trainer/trainer.py#L1576) is initialized.

Reply:
@ar-nowaczynski  Can you path the diff from @malay-nagda' PR #10125  to check if that fixes that issue ? 
and if so, close this ?

Reply:
Shortly after creating this PR, I discovered the problem on my side can be easily fixed by adding to config:
```yaml
exp_manager:
  log_tflops_per_sec_per_gpu: False
```
I'm willing to close this PR.
Do you agree to close this PR?

Reply:
Though that works, #10125 is a right fix. 

As you already have a workaround, you could wait for #10125 to be merged and check if fixes without that config. 

After #10125, this config should be a no-op. (One less config)

Reply:
Agree. Closing.